pyproject.toml

```diff
@@ -14,7 +14,7 @@ dependencies = [
     "addfips>=0.4,<0.5",
     "alembic>=1.10.3,<1.13",
     "catalystcoop.dbfread>=3.0,<3.1",
-    "catalystcoop.ferc-xbrl-extractor==0.8.3",
+    "catalystcoop.ferc-xbrl-extractor>=1.0.0,<2",
```

Comment:
We probably want to require at least `v1.1.0` to avoid the memory issue don't we?